types/html5plus/index.d.ts

```diff
@@ -0,0 +1,15476 @@
+// Type definitions for html5plus 1.0.0
```

Comment:
is this the project's name correct? I can only find html5-plus on npm.

Reply:
Yes, the project name is correct, but this project is not on the NPM.

Reply:
I am worried about name clashes with future npm projects. Do you think html5plus is likely to be added to npm in the future? Or do you think a similar, future project might want to use the html5plus name on npm?

Reply:
This name does not conflict with future NPM projects. html5plus is the product name of dcloud.io. If a future project appears on NPM under the name html5plus, it refers to the current project. For example, the html5-plus project on NPM is a d.ts file written by a third party for this project.